### DISCUSSION TITLE
[2023-08-07] GUI Team Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#7511] [Weekly Demo Day]
* Fixed scrolling in text visualization, and the issue with "partially hidden" CB entry in some situations. #7475
* PRIVATE entries are hidden again. #7497 
* Worked with @somebody1234 on implementing the newest design on the top bar, which includes icons handled by dashboard. #7488

![Screenshot from 2023-08-07 17-16-41](https://github.com/enso-org/enso/assets/3919101/b84715ee-1cc4-44ee-b606-304860bc2a4f)

**Comment by MichaelMauderer** [comment]
*  Sync breadcrumbs and documentation panel. #7508 

![Peek 2023-08-07 13-53](https://github.com/enso-org/enso/assets/1428930/a7023a3d-618b-43ff-9621-6a47906e46a3)

*  Refactor icons and breadcrumbs into separate crate. #7419 
*  Fix broken text vis layout on slow loading #7429 (fixing it again)
* Updated "Documentation panel redesign" #7372 
![image](https://github.com/enso-org/enso/assets/1428930/ac36d3fd-15f3-4275-8aa8-9e0fa7592845)

**Comment by kazcw** [comment]
- Use new font, in Rust and HTML: ![image](https://github.com/enso-org/enso/assets/1047859/cd7d2267-fb75-43e2-b32e-c1cb215be736)

**Comment by mwu-tow** [comment]
Last week:
* Landed PR that replaces statusbar and popup components with the new notifications. https://github.com/enso-org/enso/pull/7458
* Started investigation on https://github.com/enso-org/enso/issues/7218

To be done this week:
* Looking into deeplinking issue on Windows https://github.com/enso-org/cloud-v2/issues/525
* Fix the Linux->Win IDE cross-compilation https://github.com/enso-org/enso/issues/7240

**Comment by Frizi** [comment]
Last week:
- Implemented WASM panic handling. A message is displayed asking for a bug report, and project can be restarted. https://github.com/enso-org/enso/pull/7507

<img width="942" alt="Zrzut ekranu 2023-08-07 135231" src="https://github.com/enso-org/enso/assets/919491/655f8e63-35a5-4a19-805a-32b45e3623b2">

Plan for this week:
- Fixing issues found so far with new node design
